pioneer/packages/joy-proposals/src/forms/RuntimeUpgradeForm.tsx

```diff
@@ -43,10 +43,11 @@ const RuntimeUpgradeForm: React.FunctionComponent<FormInnerProps> = props => {
       <Form.Field>
         <FileDropdown<FormValues>
           setFieldValue={setFieldValue}
+          setFieldTouched={setFieldTouched}
           defaultText="Drag-n-drop WASM bytecode of a runtime upgrade (*.wasm)"
           acceptedFormats=".wasm"
           name="WASM"
-          error={errors.WASM}
+          error={touched.WASM ? errors.WASM : undefined}
```

Comment:
Yeah I had a feeling this was not correct, but as I was not super familiar with Formik I couldn't figure out this part. Late nite coding session didn't help either :)